BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/security/service/MagicFolderUtil.java

```diff
@@ -42,6 +42,19 @@ void createGroupMagicFolder(long groupId, String magicKey) {
         createMagicFolder(groupId, magicKey, "b");
     }
 
+    public String getUserMagicFolderPath(long userId, String secret) {
+        return getMagicFolderPath(userId, secret, "a");
```

Comment:
I will suggest to extract `a` and `b` to constants like `USER_FOLDER_PREFIX` so is clear the purpose of this letters